ckanext/switzerland/helpers/backend_helpers.py

```diff
@@ -294,17 +294,19 @@ def ogdch_localize_activity_item(msg):
     """localizing activity messages: this function gets an html message and
     replaces the language dict in there with the localized value
     """
-    try:
```

Comment:
Are you sure that it is safe to remove the try except block here?

Reply:
Yep - it was only catching errors stemming from trying to get the group of the regex match, when there hadn't been a match so there was no group to get.

As a general rule, it's discouraged to use a blanket `except Exception` like this, and the fewest lines of code possible should be in the try block, to make it easier to distinguish what is actually going wrong where. I took a look in the `HTMLParser` code and experimented a bit to see if there were any specific errors we should look out for, but didn't see anything.